chirpr.py

```diff
@@ -20,26 +18,36 @@ def users():
 def delete_user(user_id):
     db_access.delete_user(user_id)
     return redirect(url_for('users'))
+ 
 
-
-@app.route('/admin/user/add', methods=['POST'])
+@app.route('/user/add', methods=['POST'])
 def add_user():
     handle = request.form.get('handle')
-    db_access.add_user(handle)
+    password = request.form.get('password')
+    data = db_access.add_user(handle, password)
+    if data != None:
+        return redirect(url_for('account', success='create0'))
     return redirect(url_for('users'))
```

Comment:
It is bad practice to make successful DB actions indicated by a return value of `None`. We need to rethink this.